packages/proxy/lib/http/request-middleware.ts

```diff
@@ -185,6 +185,22 @@ const CalculateCredentialLevelIfApplicable: RequestMiddleware = function () {
   this.next()
 }
 
+const FormatCookiesIfApplicable: RequestMiddleware = function () {
+  if (this.req.headers['cookie']) {
```

Comment:
I am contemplating checking which automation client sent the value to abstract this to not apply to all the general middleware. We can't handle it at the BiDi level because it happens after the request is sent/forwarded. This just seems like a smell to me and am thinking of possible solutions

Reply:
we are going to do this with a header and strip it out to make the behavior more clear